Utilities/ci.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,7 @@
+{
+  "object": {
+    "pins": [
+    ]
+  },
+  "version": 1
+}
```

Comment:
redundant?

Reply:
We previously saw build failures without it, IIRC @neonichu already committed this on `main`?

Reply:
oh yea, I was hoping we did not need this workaround. @neonichu ?

Reply:
We do until we install a new Xcode that doesn't suffer from the problem. I haven't investigated which versions specifically hit the problem, but wasn't able to reproduce with the current release.